### ISSUE TITLE
Update the website design

### ISSUE THREAD

**Issue by msurguy** [bit101/gifloopcoder#8]
Make the website responsive and use a framework such as Bootstrap or foundation to have nice design without spending much time. 

Here's a proposed design that I made: http://msurguy.github.io/gifloopcoder.com/

![gifloopcoer](https://cloud.githubusercontent.com/assets/585833/11604879/847a1d42-9aa7-11e5-92b5-997b4d1acf05.png)

And it's on Github at https://github.com/msurguy/gifloopcoder.com/tree/gh-pages

Feel free to leave feedback.

**Comment by bit101** [comment]
Thanks. I need to look at this. It seems like a lot of code for what is essentially a single static page with a few links.

**Comment by msurguy** [comment]
@bit101 I am using Bootstrap for the layout. Bootstrap is really popular for website prototyping (number 1 css framework on the web) and most developers know how to use it which would make contributions from open source community more likely than if a custom layout framework was used.

I also used SASS version of it which essentially reduces the code you have to write if you ever need to add something custom.

**Comment by bit101** [comment]
Yeah, I understand all that. I just think it's overkill for a static landing page. Sorry if I seem a bit backwards because I don't jump on the latest, most popular technologies and frameworks. I like simplicity, minimalism and doing what works with as little extra stuff as possible.

**Comment by msurguy** [comment]
Sure, I get that you want to keep it extra lean (just bare minimum for a responsive site). I wanted the project to have traction and other contributors too and having something that almost everybody knows would've helped in that goal.

**Comment by vitorgalvao** [comment]
To give the other side of the coin, I’ll say a custom layout could be friendlier to contributors. Considering how barebones the website is (and is meant to be), it’s way easier to find and fix an error than to immerse yourself in a framework (just look at how small the CSS for the current version is). I particularly don’t see a case for it since this homepage will seldom change; contributions aren’t really a concern.

Finally, there are already bugs in this new layout:
![image](https://cloud.githubusercontent.com/assets/1699443/12886149/49b0aaec-ce63-11e5-9984-9e42fd1dce0f.png)

And it loads way slower compared to the current version (loads instantly).

Though I do agree the current version is ugly (and perhaps will even shy you away from the program if you don’t already know what it is), I don’t think a framework is the solution.

**Comment by bit101** [comment]
I agree the current version is ugly too. In fact, I was just looking at it
today and it made me a little sick. :) Now that I'm at a pretty good point
on the app itself, I can focus on stuff like this.

On Mon, Feb 8, 2016 at 7:59 AM, Vítor Galvão <notifications@github.com>
wrote:

> To give the other side of the coin, I’ll say a custom layout could be
> friendlier to contributors. Considering how barebones the website is (and
> is meant to be), it’s way easier to find and fix an error than to immerse
> yourself in a framework (just look at how small the CSS for the current
> version is). I particularly don’t see a case for it since this homepage
> will seldom change; contributions aren’t really a concern.
>
> Finally, there are already bugs in this new layout:
> [image: image]
> <https://cloud.githubusercontent.com/assets/1699443/12886149/49b0aaec-ce63-11e5-9984-9e42fd1dce0f.png>
>
> And it loads way slower compared to the current version (loads instantly).
>
> Though I do agree the current version is ugly (and perhaps will even shy
> you away from the program if you don’t already know what it is), I don’t
> think a framework is the solution.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bit101/gifloopcoder/issues/8#issuecomment-181356155>.
>



-- 
---| Keith Peters
---| kp@bit-101.com
---| www.bit-101.com

**Comment by msurguy** [comment]
@bit101 great, let me know if you need any help!

**Comment by msurguy** [comment]
@vitorgalvao the template that I made was done in 2 hours total. Of course it has bugs but it looks a little more pleasant than the current site, would you agree?

As far as the loading time, it is because there are 3 large GIFs on my demo page.
See screenshot below.
![screen shot 2016-02-09 at 9 46 34 am](https://cloud.githubusercontent.com/assets/585833/12925227/2401db86-cf12-11e5-946b-7f97c9c2b27a.png)
